unordered_associative_task_1/student/include/impl.h

```diff
@@ -32,4 +46,21 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist){
+
+    if (first_playlist.size() != second_playlist.size()){
+        return false;
+    }
+
+    std::unordered_set <Song, SongHash> first_playlist_set(first_playlist.begin(), first_playlist.end());
+    std::unordered_set <Song, SongHash> second_playlist_set(second_playlist.begin(), second_playlist.end());
```

Comment:
will it work for following case?
{"1", "1", "2"}
{"2", "2", "1"}

unique songs are "1" and "2", size is equal, but the amount of tracks differs